.devcontainer/Dockerfile

```diff
@@ -23,21 +23,13 @@
 FROM ubuntu:22.04
 RUN apt-get update
 
-# Shell
+# OS Packages
 
 RUN apt-get install --assume-yes \
   "bash=5.*" \
+  "build-essential=12.*" \
   "curl=7.*" \
   "git=1:2.*" \
-  "zsh=5.*"
-
-COPY ./scripts/ /root/
```

Comment:
The two .zshrc files should stay, and be copied, so that developers using the devcontainer get hermit accessible on their path.